base/dsfs/compute_fields.go

```diff
@@ -20,9 +20,10 @@ import (
 type computeFieldsFile struct {
 	*sync.Mutex
 
-	fs qfs.Filesystem
-	pk crypto.PrivKey
-	sw SaveSwitches
+	fs  qfs.Filesystem  // filesystem we're writing to
+	pub event.Publisher // bus to publish progress events to
```

Comment:
Can we rename this to `bus` or `publisher`, seeing it next to `pk crypto.PrivKey` threw me off a bit.